src/com/theironyard/novauc/Main.java

```diff
@@ -1,8 +1,40 @@
 package com.theironyard.novauc;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+
 public class Main {
 
-    public static void main(String[] args) {
-	// write your code here
+
+    public static void main(String[] args) throws FileNotFoundException {
+        ArrayList<Person> people = new ArrayList<>();
+        HashMap<String, ArrayList<Person>> listMap = new HashMap<>();
+
+
+        File f = new File("people.csv");
+        Scanner scanner = new Scanner(f);
+        while (scanner.hasNext()){
+            String line = scanner.nextLine();
+            String[] columns = line.split(",");
+            Person person = new Person(String.valueOf(columns[0]), columns[1], columns[2],columns[3], columns[4],columns[5]);
+            people.add(person);
+        }
+
```

Comment:
Ok next step is you need to add the person to the arraylist that maps to their country.  So you need to make a decision.  If their country is in the hashmap already add them to that arraylist.  If not add the country and new arraylist to the hashmap then add the person to that arraylist.  Then you need a toString override so the map will print nicely.  Then print the hashmap and save it to a json file.